src/looprestoration.rs

```diff
@@ -726,3 +726,223 @@ pub(crate) unsafe extern "C" fn selfguided_filter<BD: BitDepth>(
         }
     };
 }
+
+// TODO(randomPoison): Temporarily pub until init logic is deduplicated.
+pub(crate) unsafe extern "C" fn sgr_5x5_c<BD: BitDepth>(
```

Comment:
```suggestion
pub(crate) unsafe extern "C" fn sgr_5x5_c_erased<BD: BitDepth>(
```

Did you want to mark them erased, too?

Reply:
Ah yeah, let's keep the naming convention consistent.